### DISCUSSION TITLE
How long after release is webassembly build published on npm/unpkg?

### DISCUSSION THREAD

**Discussion by BStringhamVRSK** [google/filament#4148] [Q&A] [answered]
We are using unpkg to get Filament webassembly builds.  It appears that the latest available is 1.10.1.  Are these automatically posted when releases are issued?
We are looking for some fixes that are included in 1.10.3, but it doesn't appear to be available, yet.
Thanks.

**Comment by prideout** [comment, marked as answer]
It's an informal process, I update the npm registry every few weeks, I'll go ahead and do so now.

**Reply by BStringhamVRSK** [reply]
That's great!  Thank you.

**Comment by prideout** [comment]
Looks like our suzanne demo has regressed, let me take a look at that before I publish the 1.10.3 build.

**Comment by prideout** [comment]
v1.10.3 is now available on npm (includes a cherry pick for the suzanne regression)

**Reply by BStringhamVRSK** [reply]
@prideout
Could I request another update to the npm registry for the latest WebGL build (1.11.0)?
So far as I can tell, it appears that 1.10.3 is the latest available.

**Reply by prideout** [reply]
Sure, I'm out of town right now but I'll try to do this soon.

**Reply by BStringhamVRSK** [reply]
Thanks!  I really appreciate it.

**Reply by prideout** [reply]
Done!

**Reply by BStringhamVRSK** [reply]
You are a gentleman and a scholar.  Thank you.
